charts/signoz/templates/otel-collector/deployment.yaml

```diff
@@ -55,22 +55,15 @@ spec:
       {{- end }}
       securityContext:
         {{- toYaml .Values.otelCollector.podSecurityContext | nindent 8 }}
+      # todo: add k8s-wait-for initContainer here
+      # this initContainer waits for the schema migrator job to finish
       initContainers:
-        {{- if .Values.otelCollector.initContainers.init.enabled }}
-        - name: {{ include "otelCollector.fullname" . }}-init
-          image: {{ include "otelCollector.initContainers.init.image" . }}
-          imagePullPolicy: {{ .Values.otelCollector.initContainers.init.image.pullPolicy }}
-          env:
-            {{- include "snippet.clickhouse-credentials" . | nindent 12 }}
-          {{- with .Values.otelCollector.initContainers.init.command }}
-          command:
-            - sh
-            - -c
-            - until wget --user "${CLICKHOUSE_USER}:${CLICKHOUSE_PASSWORD}" --spider -q {{ include "clickhouse.httpUrl" $ }}{{ .endpoint }}; do echo -e "{{ .waitMessage }}"; sleep {{ .delay }}; done; echo -e "{{ .doneMessage }}";
-          {{- end }}
-          resources:
-            {{- toYaml .Values.otelCollector.initContainers.init.resources | nindent 12 }}
-        {{- end }}
```

Comment:
@dhawal1248 Can you change this back? We will need this in otel-collector to wait for clickhouse to be ready.